repository/src/main/java/com/rudderstack/android/repository/RudderDatabase.kt

```diff
@@ -65,7 +69,14 @@ object RudderDatabase {
         databaseCreatedCallback: ((SQLiteDatabase?) -> Unit)? = null,
         databaseUpgradeCallback: ((SQLiteDatabase?, oldVersion: Int, newVersion: Int) -> Unit)? = null
     ) {
-        commonExecutor = executorService ?: Executors.newCachedThreadPool()
+        Executors.newCachedThreadPool()
```

Comment:
`Executors.newCachedThreadPool()` value is not set in any variable. Should we remove this?